esky/bdist_esky/f_py2exe.py

```diff
@@ -22,7 +22,11 @@
 import ctypes
 
 
-from py2exe.distutils_build_exe import py2exe
+try:
+  from py2exe.build_exe import py2exe
+except:
```

Comment:
Thanks @rhlobo!  Can you please tweak this to catch the `ImportError` class specifically rather than usinga bare `except:`?